my-webxr-app/src/components/DataPointMenu.tsx

```diff
@@ -32,8 +33,11 @@ export default function DataPointMenu(
           {`${selectedDataPoint?.color ?? '-'}\n`}
           x, y, z:
           {' '}
-          {`${selectedDataPoint?.meshProps?.position ?? '-'}\n`}
-          Column X:
+          {`${selectedDataPoint?.meshProps?.position
+            ? `${(selectedDataPoint.meshProps.position as THREE.Vector3).x.toFixed(2)}, 
+            ${(selectedDataPoint.meshProps.position as THREE.Vector3).y.toFixed(2)}, 
+            ${(selectedDataPoint.meshProps.position as THREE.Vector3).z.toFixed(2)}`
+            : '-'}\n`}
```

Comment:
Maybe the newline on line 40 is the cause of values x, y, z being printed on separate lines?

Reply:
yes, i just wasnt sure how to make it so that it didnt take it into consideration. I had to put it like this because of our line limit with the linter

Reply:
Oh ok, so getting rid of the newline character on line 40 doesn't fix the x, y, z formatting on separate lines?

Reply:
it could, I didnt give it a try, I didnt know thats how the formatting worked, was too focused on making sure it worked haha ill take a look at it

Reply:
After trying it out, that did not work, all it did was make the name of the person attach to the end of the coordinates.

Reply:
Yeah no worries! Give that a try and see if it fixes the formatting

You can also modify the Plane args to get all the text within the grey square. And there may be an additional newline on line 27, so you can also get rid of that to bump up the 'Marker' label. Hope that helps!

Other than that, the fix seems to be working good on my end! And great work with the smaller data point size 🙌